### PR TITLE
update readme example code

### PR DESCRIPTION
**Pull request by joewagner**
I found a few nits in the example code in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,17 +40,17 @@ You should also be able to use the module directly in modern browsers supporting
 
 ```typescript
 // Load module
-import init from "@tableland/sqlparser";
+import { init } from "@tableland/sqlparser";
```
Comment (carsonfarmer) [inline README.md:43]: Wait, doesn't the default import also work?

File: README.md [hunk 2]
```diff
@@ -40,17 +40,17 @@ You should also be able to use the module directly in modern browsers supporting
...
 // Parse sql statement
 const { statements, type, tables } = await sqlparser.normalize(
-  "select * FrOM fake_table_1 WHere something='nothing';"
+  "select * FrOM fake_table_1 WHere something = 'nothing';"
 );
 console.log(statements);
 console.log(type);
 console.log(tables);
-// ["select * from fake_table_1 where something = 'nothing'"]
+// ["select * from fake_table_1 where something='nothing'"]
```
Comment (carsonfarmer) [inline README.md:53]: Nice

**Review by carsonfarmer — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (joewagner) [reply README.md:43]: It works if you're using some kind of tooling that enables synthetic default imports.  If you just try to run this code natively, e.g. using node.js, you will get an error saying `TypeError: init is not a function`

Comment (joewagner) [reply README.md:43]: I tried running this out of curiosity and I had to use `import { init }`. We can leave it as the default, but maybe add a comment on how to use it?

Comment (carsonfarmer) [reply README.md:43]: Huh, that's probably some issue with our type defs. It should work. But I _much_ prefer explicit imports like this anyway.

**Review by carsonfarmer — APPROVED** [review]

**Pull request merged**